test/integration/test_timeout.py

```diff
@@ -0,0 +1,224 @@
+#  © Copyright IBM Corporation 2020.
+#  #
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  #
+#       http://www.apache.org/licenses/LICENSE-2.0
+#  #
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import time
+import unittest
+
+import requests
+from unittest.mock import Mock
+from requests import Response
+from urllib3 import Timeout
+
+from ibmcloudant.cloudant_v1 import CloudantV1
+from ibm_cloud_sdk_core.authenticators import BasicAuthenticator, IAMAuthenticator, NoAuthAuthenticator
+from ibmcloudant.couchdb_session_authenticator import CouchDbSessionAuthenticator
```

Comment:
We should follow the PEP8 convention again here:
```suggestion
import time
import unittest
from unittest.mock import Mock

from ibm_cloud_sdk_core.authenticators import BasicAuthenticator, IAMAuthenticator, NoAuthAuthenticator
import requests
from requests import Response
from urllib3 import Timeout

from ibmcloudant.cloudant_v1 import CloudantV1
from ibmcloudant.couchdb_session_authenticator import CouchDbSessionAuthenticator
```

Reply:
Changed 24bad01